statsmodels/discrete/discrete_model.py

```diff
@@ -747,21 +747,20 @@ def predict(self, params, exog=None, exposure=None, offset=None,
         #TODO: add offset tp
         if exog is None:
             exog = self.exog
-            offset = getattr(self, 'offset', 0)
+        
+        if exposure is None:
             exposure = getattr(self, 'exposure', 0)
-
         else:
-            if exposure is None:
-                exposure = 0
-            else:
-                exposure = np.log(exposure)
-            if offset is None:
-                offset = 0
+            exposure = np.log(exposure)
+
+        if offset is None:
+            offset = getattr(self, 'offset', 0)
 
+        Xb = np.dot(exog, params[:exog.shape[1]])
```

Comment:
but I don't like `X` and I don't like `beta`, and we standardized on `linpred` in GLM, and capital letters are reserved for class names.
(but there is still a lot of code with xb)

In my youth there was too much Greek that I couldn't pronounce, and I never remember what the one with an extra squiggle is. :)